src/label.styles.ts

```diff
@@ -45,6 +45,7 @@ export default [
     .tooltips {
       align-items: center;
       display: flex;
+      column-gap: var(--glide-core-spacing-base-xxs);
```

Comment:
Build will likely fail now because these properties aren't sorted alphabetically. I set you up!

Reply:
Hah, all good! Amended commit on the way